lib/hodor/environment.rb

```diff
@@ -42,24 +42,6 @@ def logger
       @logger
     end
 
-    def erb_sub(erb_body)
-      ERB.new(erb_body).result(self.instance_eval { binding })
-    end
-
-    def erb_load(filename, suppress_erb=false)
```

Comment:
Was erb_load() moved or eliminated?  I think we still need this functionality to code generate job.properties.erb files.